Lectures/Lecture5_KnowledgeDiscovery.md

```diff
@@ -1,41 +1,41 @@
 
 #### Contributors: 
 + Erik van Sebille @erikvansebille
 + Hosein Nemati
 
 ## Development goal
-__why are we covering this topic and how can this lecture help participants progress towards that learning goal __
+To gain deeper insights into the process of turning information and data into the creation of new knowledge, examples of tools that aid with some of the tasks involved (including AI-based ones), and challenges with upholding the standards of scientific integrity throughout the workflow. 
 
 ## Motivation: 
-__sketch the context of the lecture, why is this topic timely or urgent to cover, and what is desired or unwanted in the status quo.__
+Knowledge discovery is a multi-step process that starts with the collection of data and information to ultimately develop new knowledge. Students apply this directly in their master theses. As past cases of fraud and malpractices across many physics research fields have shown, retaining sound scientific practices and integrity throughout the process can sometimes be challenging, and the line between sub-optimal conduct for knowledge discovery and intentional fraud can sometimes be thin. This is becoming an increasing problem as AI-based tools that aid with knowledge discovery are becoming more powerful. They provide useful tools to alleviate burdens such as compiling background information on a topic from numerous sources but on the other hand, it is also important to be mindful of their limitations and potential misuses.  
 
 ## Vision of outcome
-__What skills and capabilities will the course participants obtain after finishing this lecture and the corresponding assignments?__
+Students will increase their awareness of useful tools for knowledge discovery that they can also use for their master thesis. They will also develop a greater appreciation for some of the challenges with creating new knowledge. 
 
 ## Embedding/community
-__please mention the main community that should be considered when studying the topics of this lecture, e.g. individual researchers, research community, employers, general public, etc. __
+The topics of this lecture pertain mainly to the research process carried out by individual researchers or research teams, tools they may use and the impacts of their conduct on the broader research community.
 
 ## Methods/exercises
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
-
++ The knowledge discovery process and AI-based tools (incl. ChatGPT)
++ guest lecture by Rens van de Schoot on ASreview: Active Learning for Systematic Reviews https://github.com/asreview/  
++ hands-on use of the ASreview tool  
++ homework assignment on investigating example cases of fraud in physics
+  
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
+- students work with the ASreview tool https://github.com/asreview/ to look up information on the topic of their master thesis
+- students work on the homework assignment on a case study of fraud in physics
```

Comment:
And when will they have to hand it in?

Reply:
changed the assignment to remove the ASreview one and added the due date for the homework